requirements.txt

```diff
@@ -2,7 +2,6 @@ lxml==4.1.1
 tqdm==4.19.4
 requests==2.18.4
 gensim==2.3.0
-tensorflow_gpu==1.4.0
```

Comment:
6. Why?

Reply:
to run the library on machines without GPU

Reply:
Ok. Anyway, I don't know who inserted this req. : )

Reply:
Should be solved more cleverly in the future.